Arduino-Bling/RobotBling/RobotBling.ino

```diff
@@ -79,17 +79,18 @@ const rgbColor rgbColor_MAGENTA_DIM(0x330033);
  */
 enum
 {
-  OFF = 0,
   //NEEDS
   BLING_COMMAND_OFF
```

Comment:
Since the bling commands are sync'd between two parties, it is good practice to declare the 1st enum item as '0'. Change this line to: 
`BLING_COMMAND_OFF = 0,`